flashcards/models.py

```diff
@@ -0,0 +1,19 @@
+from django.db import models
+
+
+class Deck(models.Model):
+    owner = models.ForeignKey(
+        'auth.User',
+        on_delete=models.CASCADE,
+    )
+
+    title = models.CharField(max_length=25)
+    last_seen_date = models.DateTimeField()
+
+
+class Flashcard(models.Model):
+    parent_deck = models.ForeignKey(Deck, on_delete=models.CASCADE)
```

Comment:
You might want to add `related_name="flashcards"` here, which will mean you can access a Deck's flashcards from the Deck instance, as well as being able to access the Deck from a Flashcard instance.